[WIP] [RHDEVDOCS-5264]: Clean up web terminal docs and add new config section (#60213)
openshift/openshift-docs: web_console/web_terminal/uninstalling-web-terminal.adoc

:_content-type: ASSEMBLY
[id="uninstalling-web-terminal"]
= Uninstalling the web terminal
include::_attributes/common-attributes.adoc[]
:context: uninstalling-web-terminal

toc::[]

> Uninstalling the {web-terminal-op} does not remove any of its custom resource definitions (CRDs) or managed resources that are created when the Operator is installed. These components must be manually uninstalled for security purposes. Removing these components also allows you to save cluster resources by ensuring that terminals do not idle when the Operator is uninstalled.

Reviewer: ```suggestion
Uninstalling the {web-terminal-op} does not remove any of the custom resource definitions (CRDs) or managed resources that are created when the Operator is installed. For security purposes, you must manually uninstall these components. By removing these components, you save cluster resources because terminals do not idle when the Operator is uninstalled.
```
- Consider editing to remove passive voice.
- Consider editing last sentence to remove `allows you to save`. Per ISG. "allows" is anthropocentric: https://www.ibm.com/docs/en/ibm-style?topic=grammar-anthropomorphism
